2-mandatory/4-stocks.js

```diff
@@ -48,7 +53,10 @@ function getAveragePrices(closingPricesForAllStocks) {
     The price change value should be rounded to 2 decimal places, and should be a number (not a string)
```

Comment:
Is the _* 1_ used to convert a string to a number?

Reply:
Yes because toFixed(2) returns a string